test/contrib/automotive/ccp.uts

```diff
@@ -903,7 +903,7 @@ assert hasattr(dto, "load") == False
 assert dto.MTA0_extension == 2
 assert dto.MTA0_address == 0x34002006
 
-= CAN Socket sr1 with dto.ansers(cro) == False
+= CAN Socket sr1 with dto.answers(cro) == False
```

Comment:
How did this work before?

Reply:
![IMG_1824](https://github.com/secdev/scapy/assets/10530980/918dc6d8-a397-448d-bd39-ad245ed1035d)

Sorry I'm dumb, but the layout was unlucky.